yarn.lock

```diff
@@ -64,6 +64,13 @@
   dependencies:
     "@types/chai" "*"
 
+"@types/chai-string@^1.4.1":
```

Comment:
I don't see anything in this PR that would require an update to `yarn.lock`.  Can you revert this?

Reply:
I take that back.  It looks like the lock file is out of sync.